src/autoscaler/testhelpers/cf.go

```diff
@@ -30,18 +30,18 @@ func GetDbVcapServices(creds map[string]string, databaseNames []string, dbType s
 // supports multiple db tags in the same vcap user provided service
```

Comment:
There is a unused func "GetStoredProcedureDbVcapServices" ..

Reply:
removed in f713090b8